lib/filewatch/tail.rb

```diff
@@ -183,7 +184,12 @@ def _read_file(path, &block)
             yield(path, line)
             @sincedb[@statcache[path]] += (line.bytesize + delimiter_byte_size)
           end
-        rescue Errno::EWOULDBLOCK, Errno::EINTR, EOFError
+        rescue EOFError
```

Comment:
small question, why only focusing on EOFError? are the others not relevant? just wondering.